server/src/course/course.controller.ts

```diff
@@ -65,15 +65,22 @@ export const getReviewsCrossListOR = async ({
     if (!crossListOR) {
       return null;
     }
-
     const reviews = await findReviewCrossListOR(crossListOR);
-    const sanitizedReviews = reviews.map((review) => {
-      const copy = review;
-      copy.user = '';
-      return copy;
-    });
 
-    return sanitizedReviews;
+    // Return only the text from each review if returnTextOnly is true
+    if (returnTextOnly) {
+      return reviews.map(review => review.text).join(" ");
+    }
+    else {
```

Comment:
In the case where returnTextOnly is false, would returning sanitizedReviews be a safe way to handle the data because it could lead to errors with the OpenAI call?